Frontend/CO.CDP.OrganisationApp/Pages/Consortium/ConsortiumEnterSharecode.cshtml.cs

```diff
@@ -60,7 +60,14 @@ public async Task<IActionResult> OnPost()
 
         try
         {
-            var shareCode = await dataSharingClient.GetSharedDataAsync(EnterSharecode);
+            var shareCode = await dataSharingClient.GetSharedDataAsync(EnterSharecode!);
+
+            if (shareCode == null)
+            {
+                ModelState.AddModelError(nameof(EnterSharecode), StaticTextResource.Consortium_ConsortiumEnterSharecode_InvalidCode);
+                return Page();
+            }
```

Comment:
Similar here - would this ever actually return null in reality? The API definition suggests that it would be throwing an ApiException here too?

```
        app.MapGet("/share/data/{sharecode}",
            [OrganisationAuthorize([AuthenticationChannel.OrganisationKey, AuthenticationChannel.OneLogin])]
        async (string sharecode,
                IUseCase<string, SupplierInformation?> useCase)
            => await useCase.Execute(sharecode)
                .AndThen(supplierInformation => supplierInformation != null ? Results.Ok(supplierInformation) : Results.NotFound()))
```